ksql-test-util/src/main/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
why is this getting sneaked in?

Reply:
It's not sneaking! It's just needed to allow mocking of `final` classes.